.ci/setup_env_centos.sh

```diff
@@ -26,26 +19,20 @@ sudo -E yum install -y epel-release
 
 # Enable priority to CentOS Base repo in order to
 # avoid perl updating issues
-if [ "$centos_version" == "8" ]; then
-	sudo echo "priority=1" | sudo tee -a /etc/yum.repos.d/CentOS-Base.repo
-	sudo -E yum -y clean all
-fi
+sudo echo "priority=1" | sudo tee -a /etc/yum.repos.d/CentOS-Base.repo
+sudo -E yum -y clean all
 
 echo "Update repositories"
 sudo -E yum -y update
 
-if [ "$centos_version" == "8" ]; then
-	echo "Enable PowerTools repository"
-	sudo -E yum install -y yum-utils
-	sudo yum-config-manager --enable PowerTools
-fi
+echo "Enable PowerTools repository"
+sudo -E yum install -y yum-utils
+sudo yum-config-manager --enable PowerTools
 
 echo "Install chronic"
 sudo -E yum -y install moreutils
 
-if [ "$centos_version" == "8" ]; then
-	chronic sudo -E yum -y install pkgconf-pkg-config
-fi
+chronic sudo -E yum -y install pkgconf-pkg-config
```

Comment:
The build image step on OpenShift CI is currently carried out on Centos 7. So this change is going to break it.

More in details, the `github.com/kata-containers/tests/.ci/openshift-ci/buildall_install.sh` will be executed in a Centos 7 container.

Reply:
@wainersm, I've dropped the second patch.
I'm sorry, I didn't realise it'd affect the OpenShift CI, my bad.

Reply:
np @fidencio , because the OpenShift CI is a separate folder/code those things happens. I will on the future enable testing of PR on openshift ci so that we will be able to catch that kind of regressions.